pkg/license/license_test.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2025 Chainguard, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package license
+
+import (
+	"context"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	apkofs "chainguard.dev/apko/pkg/apk/fs"
+	"chainguard.dev/melange/pkg/config"
+)
+
+func TestFindLicenseFiles(t *testing.T) {
+	// Create a temporary directory for testing
+	tempDir, err := os.MkdirTemp("", "license_test")
```

Comment:
could we use `t.TempDir()` here?

Reply:
Whoops, missed that one! I converted many test cases from my earlier usage of MkdirTemp, but missed this one.